docs/concepts/automations.md

```diff
@@ -104,28 +104,32 @@ Specify a name and, optionally, a description for the automation.
 
 ## Custom triggers
 
-Custom triggers allow advanced configuration of the conditions on which an automation executes its actions. Several custom trigger fields accept values that end with trailing wildcards, like `"prefect.flow-run.*"`.
+When you need a trigger that doesn't quite fit the common templates in the Automations section of the UI, you can define an automation's trigger in JSON.  With custom triggers, you have access to the full capabilities of Prefect's automation system - allowing you to react to many kinds of events and metrics changing with your workspace.
```

Comment:
```suggestion
When you need a trigger that doesn't quite fit the common templates in the Trigger Type drop down on the Automations page in the UI, you can define an automation's trigger in JSON by selecting a Custom Trigger Type.  With custom triggers, you have access to the full capabilities of Prefect's automation system - allowing you to react to many kinds of events and metrics changing with your workspace.
```

I also kinda feel like a picture could say 1000 words here maybe instead of my suggestion

Reply:
Yep totally agree.  I'll drop a pic of the UI here but I think I'll keep my original phrasing